internal/pkg/skuba/addons/dex.go

```diff
@@ -161,8 +161,12 @@ kind: Deployment
 metadata:
   name: oidc-dex
   namespace: kube-system
+  labels:
+    app: oidc-dex
+    caasp.suse.com/skuba-addon-ha: "true"
```

Comment:
Follow addon RFC https://github.com/SUSE/caasp-rfc/blob/master/accepted/0043-addon-upgrade.md#resource-annotations

1. Does it better change the naming like `addon.caasp.suse.com/skuba-ha` or `addon.caasp.suse.com/skuba-ha`?

2.  Does it better change the location under?
```
spec:
  template:
    metadata:
      annotations:
```